packages/beacon-node/src/network/processor/gossipQueues/indexed.ts

```diff
@@ -18,28 +29,33 @@ import {GossipQueue, IndexedGossipQueueMinSizeOpts} from "./types.js";
  */
 export class IndexedGossipQueueMinSize<T extends {indexed?: string}> implements GossipQueue<T> {
```

Comment:
if not already you should collect this metrics when the node is scraped:
- Total count of unique data strings in the cache
- Total count of individual attestations in the cache
- Current histogram (reset, iterate, observe; like we do in the peer manager) of how long each data has been in the queue since addition
- Histogram of the age of a data when it is picked by the processor

Reply:
added missing metrics 👍